arches/app/src/components/ControlledListManager/ControlledListTable.vue

```diff
@@ -0,0 +1,397 @@
+                ...item.labels
+                    .filter(
+                        (label) =>
+                            label.language === selectedLanguage.value &&
+                            label.valuetype === "prefLabel"
+                    )
+                    .map((label) => label.value),
+            ].join(", "),
+            altLabels: [
+                ...item.labels
+                    .filter(
+                        (label) =>
+                            label.language === selectedLanguage.value &&
+                            label.valuetype === "altLabel"
+                    )
+                    .map((label) => label.value),
+            ].join(", "),
+        };
+    });
+});
+</script>
+
+<template>
+    <ListHeader
+        :displayed-list="props.displayedList"
+        :is-item-editor="false"
+    />
+
+    <div
+        v-if="props.displayedList"
+        class="list-editor-container"
+    >
+        <ListCharacteristics
+            :displayed-list="props.displayedList"
+            :editable="false"
+        />
+        <div
+            class="items"
+            style="height: 50vh"
+        >
+            <h3 style="margin-top: 4rem; margin-left: 0">
+                Items ({{ props.displayedList.items.length }})
+            </h3>
+            <div style="height: 100%">
+                <div class="controls">
+                    <SplitButton
+                        class="button language-selector"
+                        :label="`Language - ${languageMap?.[selectedLanguage]}`"
+                        :model="languageDropdownItems"
+                        raised
+                        :pt="{
+                            button: {
+                                root: {
+                                    style: {
+                                        background: 'var(--gray-700)',
+                                        border: '1px solid var(--gray-700)',
+                                    },
+                                },
+                            },
+                            menubutton: {
+                                root: {
+                                    style: {
+                                        background: 'var(--gray-800)',
+                                        border: '1px solid var(--gray-800)',
+                                    },
+                                },
+                            },
+                            menu: {
+                                root: { class: 'language-item' },
+                            },
+                        }"
+                    />
+                    <Button
+                        class="button manage-list"
+                        label="Manage List"
+                        raised
+                        @click="() => setEditing(true)"
+                    />
+                </div>
+                <!-- TreeTable exists, but DataTable has better support for reordering -->
```

Comment:
Will do exactly this in the next iteration, and use `TreeTable` once we land on a separate UX for reordering (#10600).